src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -246,12 +246,12 @@ trait AttributeComponent {
                 set a.value_string=ta.value_string, a.value_number=ta.value_number, a.value_boolean=ta.value_boolean, a.value_entity_ref=ta.value_entity_ref
                 where ta.owner_id is not null and a.owner_id in """, ownerIdTail(ownerIds)).as[Int]
       def insertIntoMasterAction(ownerIds: Seq[OWNER_ID]) =
-        concatSqlActions(sql"""insert into #${baseTableRow.tableName}(name,value_string,value_number,value_boolean,value_entity_ref,list_index,owner_id)
+        concatSqlActions(concatSqlActions(sql"""insert into #${baseTableRow.tableName}(name,value_string,value_number,value_boolean,value_entity_ref,list_index,owner_id)
                 select ta.name,ta.value_string,ta.value_number,ta.value_boolean,ta.value_entity_ref,ta.list_index,ta.owner_id
                 from #${baseTableRow.tableName}_TEMP ta
                 left join #${baseTableRow.tableName} a
-                on (a.name,a.owner_id,a.list_index)<=>(ta.name,ta.owner_id,ta.list_index)
-                where a.owner_id is null and ta.owner_id in """, ownerIdTail(ownerIds)).as[Int]
+                on (a.name,a.owner_id,a.list_index)<=>(ta.name,ta.owner_id,ta.list_index) and a.owner_id in """, ownerIdTail(ownerIds)),
```

Comment:
before there was no limit on a.owner_id, so it was doing a full table scan (100m records in dev). this limits us to only reading the rows that have entities that we're dealing with

explain plan courtesy of doug:

`+----+-------------+-------+-------+----------------------+----------------------+---------+------+------+-----------------------------------------------------------------------------+
| id | select_type | table | type  | possible_keys        | key                  | key_len | ref  | rows | Extra                                                                       |
+----+-------------+-------+-------+----------------------+----------------------+---------+------+------+-----------------------------------------------------------------------------+
|  1 | SIMPLE      | ta    | ALL   | NULL                 | NULL                 | NULL    | NULL |    1 | Using where; Using temporary                                                |
|  1 | SIMPLE      | a     | range | UNQ_ENTITY_ATTRIBUTE | UNQ_ENTITY_ATTRIBUTE | 8       | NULL |    4 | Using where; Not exists; Using index; Using join buffer (Block Nested Loop) |
+----+-------------+-------+-------+----------------------+----------------------+---------+------+------+-----------------------------------------------------------------------------+`